macros/src/lib.rs

```diff
@@ -396,7 +416,17 @@ pub fn exception(args: TokenStream, input: TokenStream) -> TokenStream {
             let tramp_ident = Ident::new(&format!("{}_trampoline", f.sig.ident), Span::call_site());
             let ident = &f.sig.ident;
 
+            let (ref cfgs, ref attrs) = extract_cfgs(f.attrs.clone());
+
+            if containts_blacklist_attrs(cfgs) || containts_blacklist_attrs(attrs) {
+                return parse::Error::new(Span::call_site(), "This attribute is not allowed")
```

Comment:
```suggestion
                return parse::Error::new(Span::call_site(), "this attribute is not allowed")
```

This should either point at the attribute in question, or mention the attribute in the error message. Right now it will point at the cortex-m-rt macro.

Reply:
Fixed this